src/venv_manager.rs

```diff
@@ -172,21 +173,22 @@ impl VenvManager {
     fn create_venv(&self) -> Result<(), Error> {
         let parent_venv_path = &self.paths.venv.parent();
         if parent_venv_path.is_none() {
-            return error::new("venv_path has no parent");
+            return Err(Error::Other {
+                message: "venv_path has no parent".to_string(),
+            });
         }
         let parent_venv_path = parent_venv_path.unwrap();
```

Comment:
You can also use combinators on Options if you want, not just Results!
The 7 lines above can be:

```rust
let parent_venv_path = &self.paths.venv.parent().ok_or(Error::Other {
    message: "venv_path has no parent".to_string(),
})?;
```